tracks/ruby/exercises/series/mentoring.md

```diff
@@ -0,0 +1,49 @@
+New concepts: Iterating over an array (preferably with `each_cons`, avoid `each_with_index`); `.chars` to split a string; (private) `attr_reader`; raising an Argument Error; inline if-statement 
+
+### Minimal solution for approval
+
+```ruby
+
+class Series
+
+  def initialize(serie)
```

Comment:
If we are working with a series and we ask for a serie, since they have the same meaning, why do we need the class?  It is self referential.  It says to me "Hey, I will build a series for you if you give me a series."  Instead, we want a collection of numbers as a string.  I think we can ask for that.

Reply:
I don't know.  I only knew what was being asked for after reading the README for the exercise.  The name 'serie' was confusing me because of that self reference, so I had to reach for external documentation instead of the source code.  A 'collection of digits' is what is important to receive, the tests will encourage how that is implemented from there.

Reply:
`digits.chars` maybe?  Yes, it is a method name, but  perhaps `numbers` would be better.  I think `numerals` would work, but generally these represent the symbols used, rather than numbers,  Then again, it is what we want, a collection of those symbols, to break out into the series, not necessarily treating the collection of the digits as a single number, and not at this point.